spec/requests/api/framework_responses_controller_update_spec.rb

```diff
@@ -6,6 +6,10 @@
   describe 'PATCH /framework_responses/:framework_response_id' do
     include_context 'with supplier with spoofed access token'
 
+    subject(:patch_response) do
+      patch "/api/v1/framework_responses/#{framework_response_id}?include=#{includes}", params: framework_response_params, headers: headers, as: :json
```

Comment:
no `v1` required in the URL - we want to deprecate those

Reply:
Done